src/NHSD.BuyingCatalogue.API/Startup.cs

```diff
@@ -60,7 +60,6 @@ public void ConfigureServices(IServiceCollection services)
 
             services
                 .AddSingleton<ISettings>(settings)
-                .AddTransient(typeof(IPipelineBehavior<,>), typeof(LoggingBehaviour<,>))
```

Comment:
Any reason why this got removed?

Reply:
Logging behaviour was removed, as it wasnt being used anymore. There was no need for the transient to be used